internal/service/verify/resource_voice_phrase_content.go

```diff
@@ -0,0 +1,422 @@
+		},
+		"UpdateVoicePhraseContent",
+		framework.DefaultCustomError,
+		sdk.DefaultCreateReadRetryable,
+		&response,
+	)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Update the state to save
+	state = plan
+
+	// Save updated data into Terraform state
+	resp.Diagnostics.Append(state.toState(response)...)
+	resp.Diagnostics.Append(resp.State.Set(ctx, state)...)
+}
+
+func (r *VoicePhraseContentResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	var data *voicePhraseContentResourceModel
+
+	if r.client == nil {
+		resp.Diagnostics.AddError(
+			"Client not initialized",
+			"Expected the PingOne client, got nil.  Please report this issue to the provider maintainers.")
+		return
+	}
+
+	// Read Terraform prior state data into the model
+	resp.Diagnostics.Append(req.State.Get(ctx, &data)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Run the API call
+	resp.Diagnostics.Append(framework.ParseResponse(
+		ctx,
+
+		func() (any, *http.Response, error) {
+			r, err := r.client.VoicePhraseContentsApi.DeleteVoicePhraseContent(ctx, data.EnvironmentId.ValueString(), data.VoicePhraseId.ValueString(), data.Id.ValueString()).Execute()
+			return nil, r, err
+		},
+		"DeleteVoicePhrase",
+		framework.CustomErrorResourceNotFoundWarning,
+		sdk.DefaultCreateReadRetryable,
+		nil,
+	)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+func (r *VoicePhraseContentResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	splitLength := 3
+	attributes := strings.SplitN(req.ID, "/", splitLength)
+
+	if len(attributes) != splitLength {
+		resp.Diagnostics.AddError(
+			"Unexpected Import Identifier",
+			fmt.Sprintf("invalid id (\"%s\") specified, should be in format \"environment_id/voice_phrase_id/voice_phrase_content_id\"", req.ID),
+		)
+		return
+	}
+
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("environment_id"), attributes[0])...)
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("voice_phrase_id"), attributes[0])...)
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("id"), attributes[1])...)
+}
+
+func (p *voicePhraseContentResourceModel) expand() (*verify.VoicePhraseContents, diag.Diagnostics) {
+	var diags diag.Diagnostics
+
+	data := verify.NewVoicePhraseContentsWithDefaults()
+
+	data.SetId(p.Id.ValueString())
+
+	if !p.VoicePhraseId.IsNull() && !p.VoicePhraseId.IsUnknown() {
+		data.SetVoicePhrase(*verify.NewVoicePhraseContentsVoicePhrase(p.VoicePhraseId.ValueString()))
+	}
+
+	if !p.Content.IsNull() && !p.Content.IsUnknown() {
+		data.SetContent(p.Content.ValueString())
+	}
+
+	if !p.Locale.IsNull() && !p.Locale.IsUnknown() {
+		data.SetLocale(p.Locale.ValueString())
+	}
+
+	if !p.CreatedAt.IsNull() && !p.CreatedAt.IsUnknown() {
```

Comment:
`CreatedAt` and `UpdatedAt` shouldn't be handled as an input parameters in the `expand` method